products/storage/api.yaml

```diff
@@ -107,17 +168,78 @@ objects:
               item_type: Api::Type::String
       - !ruby/object:Api::Type::Array
         name: 'defaultObjectAcl'
-        item_type: !ruby/object:Api::Type::NestedObject
```

Comment:
I moved `description + input: true` before the list of properties because it's much easier to read that way. That's why the diff looks like this.